apps/03TicTacToe.js

```diff
@@ -1,17 +1,21 @@
-//cleanup code
-//re-write program on whiteboard
-//re-write program in code
-//flesh out program in html/css
-//colt solution
+//question
+  //scope
+    //how does printBoard() have access to board variable inside of resetBoard()
+      //would think printBoard() would only have access to global 'var board'
+        //such that it would not work to call resetBoard()
+          //and then call printBoard()
+            //and expect printBoard()
+              //to print a fresh board
+                //rather than an old board that has been played on already
```

Comment:
Hi Judah,

Lets talk about this question in class today. printBoard() does not have access to a variable in resetBoard, only the global var. If this is the code you are referring to:

'''javascript
function resetBoard(){
  board = [
    [' ', ' ', ' '],
    [' ', ' ', ' '],
    [' ', ' ', ' ']
  ];

  playerTurn = 'X';

}

function printBoard() {
  console.log('   0  1  2');
  console.log('0 ' + board[0].join(' | '));
  console.log('  ---------');
  console.log('1 ' + board[1].join(' | '));
  console.log('  ---------');
  console.log('2 ' + board[2].join(' | '));
}
'''

both resetBoard and printBoard are accessing the global var.